src/lib/functions/synchronous.js

```diff
@@ -36,14 +38,26 @@ const handleSynchronousFunction = function (err, result, response) {
   response.end()
 }
 
-const formatLambdaLocalError = (err) => `${err.errorType}: ${err.errorMessage}\n  ${err.stackTrace.join('\n  ')}`
+const formatLambdaLocalError = (err) =>
+  JSON.stringify({
+    errorType: err.errorType,
+    errorMessage: err.errorMessage,
+    trace: err.stackTrace,
+  })
```

Comment:
[sand] For readability, multiline arrow functions should be wrapped in braces with a return unless they start on the same like as the arrow.

Reply:
@sean-roberts our global linting rules enforce the opposite.

![Screenshot 2022-03-28 at 15 16 46](https://user-images.githubusercontent.com/4162329/160418062-36ddd742-b182-4e57-a8ec-fca58bce08e5.png)

Reply:
grrrroosss :) Can you start it on the above line? 
```suggestion
const formatLambdaLocalError = (err) => JSON.stringify({
    errorType: err.errorType,
    errorMessage: err.errorMessage,
    trace: err.stackTrace,
  })
```

Reply:
Formatting is handled automatically by Prettier. 😄

Reply:
yeah I figured, we should look at allowing that but this is up to the team. It's certainly a non-traditional code approach that makes it harder to read.